librz/core/visual.c

```diff
@@ -2003,11 +2003,13 @@ static bool insert_mode_enabled(RzCore *core) {
 	case 'e':
 	case 'f':
 		if (__nib != -1) {
-			rz_core_cmdf(core, "wx %c%c @ $$+%d", __nib, ch, core->print->cur);
+			char hexpair[3] = { __nib, ch, 0 };
+			rz_core_write_hexpair(core, core->offset + core->print->cur, hexpair);
 			core->print->cur++;
 			__nib = -1;
 		} else {
-			rz_core_cmdf(core, "wx %c. @ $$+%d", ch, core->print->cur);
```

Comment:
Is `wx 9.` even a thing???

Reply:
Nope, it returns the error.

Reply:
Should I remove this line?

Reply:
Yes, I think this line can be removed. Keep the `__nib = ch` part, that's necessary.
In visual mode you can write at minimum 1 byte.